calculator.go

```diff
@@ -1,13 +1,28 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import "fmt"
+
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
-return a + b
+	return a + b
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
 func Subtract(a, b float64) float64 {
-	return b - a
+	return a - b
+}
+
+// Multiply takes two numbers as parameters, and returns a single number representing the result.
+func Multiply(a, b float64) float64 {
+	return a * b
+}
+
+// Divide takes two numbers as parameters, and returns first divide by second representing the result.
+func Divide(a, b float64) (float64, error) {
+	if b == 0 {
+		return 0, fmt.Errorf("division by zero is undefined")
```

Comment:
If we're not interpolating any data, then we don't need `fmt.Errorf`—we can use `errors.New` instead.